media/css/base/page-not-found.scss

```diff
@@ -70,15 +70,17 @@ main {
     .logo-mozilla {
         display: inline-block;
         height: 50px;
-        transform: translateY(-6px);
+        width: 141px;
+        padding-left: 30px;
+        transform: translateY(-2px);
 
         &:after {
             @include bidi(((float, right, left),));
             background: $color-marketing-gray-30;
             content: '';
             height: 50px;
```

Comment:
I think if we increase this to `60px` it'll look better (the dividing line will be slightly taller than the two logos).

Reply:
Will look into it